packages/suite-desktop-core/src/modules/auto-updater.ts

```diff
@@ -236,8 +242,27 @@ export const init: Module = ({ mainWindow, store }) => {
         logger.info(SERVICE_NAME, `New feed url: ${feedURL}`);
     });
 
+    ipcMain.on('update/set-automatic-update-enabled', (_, value = true) => {
+        logger.info(SERVICE_NAME, `set-automatic-update-enabled: ${value ? 'true' : 'false'}`);
+
+        mainWindow.webContents.send('update/set-automatic-update-enabled', value);
+        const settings = store.getUpdateSettings();
+        store.setUpdateSettings({ ...settings, isAutomaticUpdateEnabled: value });
+        isAutomaticUpdateEnabled = value;
+
+        if (!isAutomaticUpdateEnabled) {
+            // In case
+            //      1) the auto-update was enabled
+            //      2) update is probably already downloaded
+            //      3) user wants do disable auto-update and PREVENT the downloaded update from installing
+            //
+            // We have to disable auto update so it won't get installed.
+            autoUpdater.autoInstallOnAppQuit = false;
```

Comment:
This is important :point_up: . This is the mechanic how to prevent the suite from updating when it already downloaded but user wants to disable silent update!